autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/on_server_notif.rs

```diff
@@ -43,11 +43,16 @@ impl WebPushClient {
     /// Send a Direct Push Notification to this user
     fn notif(&mut self, notif: Notification) -> Result<ServerMessage, SMError> {
         trace!("WebPushClient::notif Sending a direct notif");
+        // The notification we return here is sent directly to the client.
+        // No reliability state is recorded.
+        let response = notif.clone();
```

Comment:
Again, being a bit more explicit about the function here.
Technically, we want the `notif` to be consumed by the `unacked_direct_notifs` list since that's where the state goes. The return value is sent to the client, so it should be a copy of the original `notif` since no state data is recorded.